### ISSUE TITLE
cant change prefix color for biome announcements

### ISSUE THREAD

**Issue by kushuser** [Sinkazo/CustomBiomeAnnounce#1]
trying to edit the color for the "Entered biome:" msg but dont see a config option

**Comment by Sinkazo** [comment]
Hello sir! 

Thank you for no leave a bad rating. I appreciate it!

The option is in the config.yml. Yo can use "/cba reload" the without need  to restart your server to aplly the changes

# Configuration for sending a message in chat when entering a new biome
message:
  enabled: true
  text: "Entered biome: %biome%"

**Comment by Sinkazo** [comment]
Hi Sir.

Thank you for no leave a bad rating. 

I just fixed it! Download new jar and replace just the jar on your plugins folder

**Comment by kushuser** [comment]
![image](https://github.com/user-attachments/assets/747edfe4-f39c-4f77-b379-9cdbc58f0cb9)
i mean the Entered Biome part

**Comment by Sinkazo** [comment]
Yup, that was fixed. Try using new version on spigot or here and replace the jar

**Comment by kushuser** [comment]
i did use the new version, it dosent let me use color codes in the entered biome prefix, somthing is overriding it

**Comment by kushuser** [comment]
thanks for the nice plugin though its a cool addition to my server

**Comment by Sinkazo** [comment]
Thank you so much, I really appreciate it. I would also be very grateful if you have the chance to rate Spigot. 
https://www.spigotmc.org/resources/1-20-1-21-custombiomeannounce-announce-player-biome-change.120133/

Btw any implementations or additions you would like to suggest go ahead! :D
